components/dashboard/src/projects/NewProject.tsx

```diff
@@ -348,49 +333,42 @@ export default function NewProject() {
                     )}
                     {loaded && noReposAvailable && isGitHub() && (<div>
                         <div className="px-12 py-20 text-center text-gray-500 bg-gray-50 dark:bg-gray-800 rounded-xl">
-                            <img src={NoAccess} title="No Access" className="m-auto mb-4" />
-                            <h3 className="text-center text-gray-600 dark:text-gray-50 pb-3 font-bold">
-                                No Access
-                            </h3>
                             <span className="dark:text-gray-400">
-                                Authorize GitHub (github.com) or select a different account.
+                                Additional authorization is required to allow access to your repositories and trigger prebuilds for every new push to any of your branches.
                             </span>
                             <br />
-                            <button className="mt-6" onClick={() => reconfigure()}>Authorize</button>
+                            <button className="mt-6" onClick={() => reconfigure()}>Configure Gitpod App</button>
                         </div>
                     </div>)}
                 </div>
 
             </div>
             {isGitHub() && (
-                    <div className="pt-3">
-                        <div className="text-gray-500 text-center w-96 mx-8">
-                            Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
-                        </div>
-                        {isGitHub() && noOrgs && (
-                            <div className="text-gray-500 mx-auto text-center">
-                                Missing organizations? <a href="javascript:void(0)" onClick={e => grantReadOrgPermissions()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Grant permissions</a>
-                            </div>
-                        )}
+                <div>
+                    <div className="text-gray-500 text-center w-96 mx-8">
+                        Repository not found? <a href="javascript:void(0)" onClick={e => reconfigure()} className="text-gray-400 underline underline-thickness-thin underline-offset-small hover:text-gray-600">Reconfigure</a>
```

Comment:
This text should only appear when with the repositorylist is showing. It is not required when we prompt for app authorization.

Reply:
And only for GitHub, right? Or all providers?

Reply:
Yes - GitHub only.